src/extension/inspire/src/main/resources/org/geoserver/inspire/available_languages.properties

```diff
@@ -5,30 +5,32 @@
 #Here have added Croatian, hrv and some EFTA country languages mentioned in view
 #and download technical guidance documents.
 #List may be updated depending on INSPIRE updates see GEOS-7388 discussion.
-bul=
-cze=
-dan=
-dut=
-eng=
-est=
-fin=
-fre=
-hrv=
-ice=
-ger=
-gle=
-gre=
-gsw=
-hun=
-ita=
-lav=
-lit=
-mlt=
-nor=
-pol=
-por=
-rum=
-slo=
-slv=
-spa=
-swe=
\ No newline at end of file
+#The List maps to the ISO codes to map the INSPIRE language request parameter to the OWS 2.0
+#AcceptLanguages one.
+bul=bg
```

Comment:
Say that the INSPIRE spec has an amendment and new languages pop up, is there a way for a user to override this property file?

Reply:
From the documentation doesn't seems that there is any defined way to override the property file ( that was already existing before this modifications).